model-api/src/main/java/edu/unc/lib/boxc/model/api/rdf/Cdr.java

```diff
@@ -132,22 +132,10 @@ public static String getURI() {
             "http://cdr.unc.edu/definitions/model#storageLocation" );
 
     /**
-     * Property which holds the streaming content filename
+     * Property which holds the StreamReaper URL for streaming
```

Comment:
While it is true in practice, i don't know that we need to call out streamreaper here since we'd use the same property for non-stream reaper urls